npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -367,19 +366,7 @@ def fetch_lerna_packages_from_path(path, nested = false)
             fetch_file_if_present(File.join(path, "npm-shrinkwrap.json"))
           ].compact
         rescue Dependabot::DependencyFileNotFound
-          matches_double_glob =
```

Comment:
Was this removed because it was a fallback, but now it should work correctly the first time?

Reply:
Yes, this cleanup should come for free after this fix since globs should be now properly expanded before they are passed to this method.